pkg/llbutil/session.go

```diff
@@ -63,7 +64,8 @@ func NewSession(ctx context.Context, opts ...SessionOption) (*session.Session, e
 	}
 
 	// By default, forward docker authentication through the session.
-	attachables := []session.Attachable{authprovider.NewDockerAuthProvider(os.Stderr)}
+	dockerConfig := config.LoadDefaultConfigFile(os.Stderr)
```

Comment:
We should probably plumb stderr at some point instead of using `os.Stderr`.

Reply:
yeah good idea, we should probably stash stdio on the context in the command impls so we can fetch them as necessary